src/control/server/config.go

```diff
@@ -398,7 +414,17 @@ func (c *Configuration) Validate() (err error) {
 		if err := srv.Validate(); err != nil {
 			return errors.Wrapf(err, "I/O server %d failed config validation", i)
 		}
-	}
 
+		validConfig, err := ValidateNetworkConfig(srv.Fabric.Provider, srv.Fabric.Interface, srv.Fabric.PinnedNumaNode)
```

Comment:
Benefit value is in the eye of the beholder.  I separated it out so that we could determine the difference between an error in the stack, versus the result of that validation effort.  That said, it's not a problem to collapse two variables down to one, and just use err.  So I've made that change.